hugo/assets/scss/base/base.scss

```diff
@@ -89,6 +93,7 @@ h6 {
 p {
     font-size: inherit;
     font-style: inherit;
+    line-height: 1.75;
```

Comment:
Should only the `p` have a smaller line-height or everything? In case of the latter you can change the line-height in the `@mixin style-text` (/mixin/typography)

Reply:
Moved line-height change to `@mixin style-text`